cmd/pool/order.go

```diff
@@ -82,6 +82,14 @@ func promptForConfirmation(msg string) bool {
 	}
 }
 
+// roundUp rounds up `n` to the nearest multiple.
+func roundUp(n, multiple btcutil.Amount) btcutil.Amount {
```

Comment:
Algo looks sound at a glance. If we want to clamp things down further, it shouldn't be too involved to create a `testing/quick` test to ensure the output of this function, given arbitrary inputs (within our range bounds) should always return something that's a multiple of our unit size. This can likely live where the unit is defined.

Reply:
Could make this a method on the `SupplyUnit` itself for easy testing.